messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/jms/ExternalConnectionFactoryAdd.java

```diff
@@ -82,7 +82,7 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
         List<String> connectorNames = Common.CONNECTORS.unwrap(context, model);
         ServiceBuilder<?> builder = context.getServiceTarget()
                 .addService(serviceName)
-                .addAliases(MessagingServices.getActiveMQServiceName(name));
+                .addAliases(JMSServices.getConnectionFactoryBaseServiceName(MessagingServices.getActiveMQServiceName("")).append(name));
```

Comment:
what would be the difference between ^^ and `JMSServices.getConnectionFactoryBaseServiceName(MessagingServices.getActiveMQServiceName(name))` ?

Reply:
it is different because name is appended at the end of the service name: *jboss.messaging-activemq.jms.cf.name* while your change will create *jboss.messaging-activemq.name.jms.cf*

Reply:
A new no-arg MessagingServices.getActiveMQServiceName() would be clearer.